modules/oidc-config-fields.adoc

```diff
@@ -34,6 +34,8 @@
 | **.VERIFIED_EMAIL_CLAIM_NAME** | String | The name of the claim that is used to verify the email address of the user.
 
 | **.PREFERRED_GROUP_CLAIM_NAME** | String | The key name within the OIDC token payload that holds information about the user's group memberships.
+
+| **.OIDC_DISABLE_USER_ENDPOINT** | Boolean | Whether to allow or disable the `user_endpoint` returned for Active Directory Federation Service (ADFS) OIDC providers.
```

Comment:
As now the **azure active directory** is changed to **azure entra ID** service, suggest use the latest name.
<img width="1512" alt="image" src="https://github.com/quay/quay-docs/assets/61186078/b5f93194-2afb-4e54-9d5d-6c70194e4baa">